ui/app/styles/components/codemirror.scss

```diff
@@ -0,0 +1,127 @@
+$dark-bright: lighten($dark, 15%);
+
+.CodeMirror {
+  height: auto;
```

Comment:
It works for this context because job definitions always have a value. I ran into the problem I think you are talking about with new job. The empty editor is still only one line tall. I'll look into the bottom margin `Infinity` trick.